lib/HireVoice/Neo4j/EntityManager.php

```diff
@@ -844,4 +856,15 @@ public function getPathFinder()
     {
         return clone $this->pathFinder;
     }
+    
+    /**
+     * @return \HireVoice\Neo4j\Meta\Property
+     */
+    public function getPrimaryKeyValue($entity)
+    {
+        $meta = $this->getMeta($entity);
+        $pk = $meta->getPrimaryKey();
+        $pkGetter = 'get'.ucfirst($pk->getName());
```

Comment:
The reflected property should allow to read the value directly from the property rather than relying on the naming convention. There used to be a lot of this, but it has since been removed.

I fail to understand the purpose of the change. Is it not possible to use @Auto on an arbitrary property?